book/scheduling.md

```diff
@@ -443,6 +453,7 @@ class JSContext:
         self, method, url, body, isasync, handle):
         # ...
         def run_load():
+            if self.discarded: return
```

Comment:
I don't think we need / want this here. This is possibly run on another thread, so ideally shouldn't read `self.discarded`. (By the way, pre-existing bug! We should save `self.tab.url` outside `run_load` to avoid cross-thread reads / to avoid a security bug if you navigate before the request thread runs?) And actually issuing the request is probably safe, since the `send` was in fact executed...